doc/getting-started/understanding-config-files.md

```diff
@@ -6,21 +6,55 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 ```json
 {
-  "Producers": [
+  "LocalProducers": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "isDomain": true,
+              "domain": {
+                "domain": "x.x.x.x",
+                "port": 3001
+              }
+            }
+          ],
+          "advertise": false
+        },
+        "valency": 1
+      }
+    ]
+  },
+  "PublicProducers": [
     {
-      "addr": "x.x.x.x",
-      "port": 3001,
-      "valency": 1
+      "publicRoots" : {
+        "addrs": [
+          {
+            "isDomain": false,
+            "addr": "y.y.y.y",
+            "port": 3002
+          }
+        ],
+        "advertise": false
+      }
     }
   ]
 }
 ```
-* This means that your node will contact node at ip `x.x.x.x` on `port 3001`.
 
-* `valency` tells the node how many connections your node should have. It only has an effect for dns addresses. If a dns address is given, valency governs to how many resolved ip addresses should we maintain active (hot) connection; for ip addresses, valency is used as a Boolean value, where `0` means to ignore the address.
+* The main difference between `LocalProducers` and `PublicProducers` is that with the former
+    you can specify different groups with different valencies. That can be useful to
+    inform your node of different targets withing a group to achieve.
```

Comment:
This only explains the difference between `LocalProducers` and `PublicProducers` from a json format point of view. `localProduers` is for peers which the node always should have as hot, such as their own block producer. `PublicProducers` represent a source of fallback peers, a source of peers to be used if peers from the ledger is disabled or unavailable.

Reply:
Thank you, I'll improve this description!